buildscripts/kokoro/android-interop.sh

```diff
@@ -16,7 +13,14 @@ export CXXFLAGS=-I/tmp/protobuf/include
 export LD_LIBRARY_PATH=/tmp/protobuf/lib
 export OS_NAME=$(uname)
 
-(yes || true) | "${ANDROID_HOME}/tools/bin/sdkmanager" --licenses
+export ANDROID_HOME=/tmp/Android/Sdk
+mkdir -p "${ANDROID_HOME}/cmdline-tools"
```

Comment:
would it make sense to remove any existing directory in case it had old contents that wouldn't be overriden.

Reply:
If it already exists, this could be a noop. But it always runs on a fresh kokoro worker, so it won't exist... I think it's okay enough as-is. Longer-term I expect we'll replace these and get off kokoro (like maybe this year).